src/main/kotlin/dev/polek/adbwifi/services/PinDeviceService.kt

```diff
@@ -19,6 +19,12 @@ class PinDeviceService : PersistentStateComponent<PinDeviceService> {
     @OptionTag(converter = PinnedDeviceListConverter::class)
     var pinnedDevices: List<PinnedDevice> = listOf()
 
+    fun addPreviouslyConnectedDevice(device: Device) {
+        if (device.connectionType == WIFI && device.address?.ip!!.isNotEmpty() && !pinnedDevices.contains(device)) {
+            pinnedDevices.add(device)
```

Comment:
`List<PinnedDevice>.add(Device)` is an extension function that doesn't add the device to the list, but instead returns a new list with the added device.
Bad naming, I'll refactor it later.

```suggestion
            pinnedDevices = pinnedDevices.add(device)
```

Reply:
Updated in: [fab323f](https://github.com/y-polek/ADB-Wi-Fi/pull/21/commits/fab323f35e9b155278901bbbad2844dbbe25ea8f)